client/src/Components/PomodoroResetBtn.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { GrPowerReset } from "react-icons/gr";
+
+const PomodoroResetBtn = () => {
```

Comment:
Instead of having multiple button components. You could have one button component that takes children as a prop and returns the children.

Reply:
Thanks Calum, thanks for the feedback, I've completed the changes and will push them wish the timer logic update.